docusaurus/docs/guides/database-lifecycle.mdx

```diff
@@ -0,0 +1,44 @@
+---
+title: Database Lifecycle Management
+sidebar_position: 30
+---
+
```

Comment:
I'd maybe link to bbolt, since that's what we actually use. They are compatible, so I think we can still say it's Bolt.
https://github.com/etcd-io/bbolt